gnomad/utils/annotations.py

```diff
@@ -2190,12 +2194,22 @@ def _create_group_dicts(
 
     ancillaryResults = {"homozygotes": overall_freq["homozygote_count"]}
 
-    if input_struct.popmax:
+    sorted_faf95 = []
+
+    for pop in FAF_POPULATIONS:
```

Comment:
not sure how this loop will impact performance

Reply:
loop was removed!